rpcs3/Emu/RSX/GL/GLTextureCache.h

```diff
@@ -624,11 +630,16 @@ namespace gl
 			baseclass::on_section_resources_destroyed();
 		}
 
-		inline bool exists() const
+		bool exists() const
 		{
 			return (vram_texture != nullptr);
 		}
 
+		bool is_managed() const
+		{
```

Comment:
why is is_managed returning true if vram_texture == nullptr? I mean, if there is no managed texture, how can we still say the section is managed? I get the feeling this is added to avoid and extra exists() check at the source, but that makes it less intuitive.

Reply:
Also, try and verify that context == framebuffer if managed_texture is empty. Might cause issues for newly created sections but you shouldn't be calling is_managed on those either.

Reply:
The reason is that "is_managed" means that a section is managed (and can be modifed) by the texture cache.
A section with `!exists()` is always "managed by the texture cache". If a section exists, then it is only "managed by the texture cache" when `managed_texture.get() != nullptr`.

This means that the texture cache can call `if(section.is_managed()) do_something();` without any issue.

The reason I didn't add the framebuffer check is because that should already be covered by the `managed_texture.get() != nullptr` check, that is the whole point. At most it would be an `AUDIT`/`ASSERT` in my opinion.

Reply:
My original meaning of managed meant that the GPU resource was managed by the section itself, thus if the managed_texture was null, there was nothing to 'manage' i.e clean up after the cache and lifetime. In this context is_managed() matches the meaning but from a different context - texture cache resource vs GPU resource. This is confusing for me, since is_managed() to me means "does this section have a managed resource?", but I honestly have no better ideas for better names.